revvy/robot/remote_controller.py

```diff
@@ -12,10 +13,84 @@
 RemoteControllerCommand = namedtuple('RemoteControllerCommand', ['analog', 'buttons', 'background_command'])
 
 
+class AutonomousModeRequest:
+    NONE   = 0
+    STOP   = 1
+    START  = 2
+    PAUSE  = 3
+    RESUME = 4
+
+    def __init__(self):
```

Comment:
All of the functions here are basically a no-op. `AutonomousModeRequest` has two responsibilities:
 - it defines the request types (one of which, NONE, is redundant)
 - and it also stores itself in itself(?) basically. So we have a class that could be a field somewhere else instead.

I'm fine with it if you want to stick to this, but you are overcomplicating the problem here.